grakn-graql/src/test/java/ai/grakn/graql/internal/printer/JsonPrinterTest.java

```diff
@@ -1,145 +0,0 @@
-/*
- * Grakn - A Distributed Semantic Database
- * Copyright (C) 2016  Grakn Labs Limited
- *
- * Grakn is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * Grakn is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
- */
-
-package ai.grakn.graql.internal.printer;
-
-import ai.grakn.concept.Attribute;
-import ai.grakn.concept.AttributeType;
-import ai.grakn.concept.ConceptId;
-import ai.grakn.concept.Rule;
-import ai.grakn.graql.Printer;
-import ai.grakn.test.rule.SampleKBContext;
-import ai.grakn.test.kbs.MovieKB;
-import ai.grakn.util.Schema;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Lists;
-import mjson.Json;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Test;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
-
-import static org.junit.Assert.assertEquals;
-
-public class JsonPrinterTest {
```

Comment:
This class still exists! If we delete the test, we should delete `JsonPrinter` too.

Reply:
Murder time!

Reply:
Cancel the murder time will try to clean this up in a different PR.